src/lib/security.js

```diff
@@ -39,3 +39,6 @@ export const devEnvOnlyExport = (unprotectedExport) => {
     return undefined;
   }
 };
+// TODO: Remove after testing
+// $FlowIssue
+export const payPayDomainRegEx = /\.paypal\.(com|cn)(:\d+)?$/; // eslint-disable-line security/detect-unsafe-regex
```

Comment:
We should remove this and use the `getPayPalDomainRegex()` method that sdk-client provides.